code/game/jobs/job/support.dm

```diff
@@ -159,7 +161,40 @@
 	back = /obj/item/mod/control/pre_equipped/mining/asteroid
 	mask = /obj/item/clothing/mask/breath
 
+/datum/job/explorer
+	title = "Explorer"
+	flag = JOB_EXPLORER
+	department_flag = JOBCAT_SUPPORT
+	job_department_flags = DEP_FLAG_SUPPLY
+	total_positions = 4
+	spawn_positions = 4
+	supervisors = "the quartermaster"
+	department_head = list("Quartermaster")
+	selection_color = "#eeddbe"
+	access = list(
+		ACCESS_MAINT_TUNNELS,
+		ACCESS_EXPEDITION,
+		ACCESS_EVA,
+		ACCESS_EXTERNAL_AIRLOCKS,
+		ACCESS_TELEPORTER,
+		ACCESS_MAILSORTING,
+	)
```

Comment:
```suggestion
		ACCESS_MAILSORTING
	)
```

Reply:
Also change from `ACCESS_MAILSORTING` to `ACCESS_CARGO`